tilavarauspalvelu/settings.py

```diff
@@ -428,6 +428,7 @@ def get_git_revision_hash() -> str:
     "": {
         "small": {"size": (250, 250), "crop": True},
         "medium": {"size": (384, 384), "crop": True},
+        "large": {"size": (728, 728)},
```

Comment:
Nope, the original ratio is needed. I havent studied the layouts of the UI for this but I guess because this is going to be some static large image to be used it for the views since the original size of the image can be whatever so the static size is more manageable.

Anywho I'll add a comment which states that it needs to save the ratio because of design.